client/src/main/webjar/authentication/authentication.component.js

```diff
@@ -1,52 +0,0 @@
-(function (angular) {
```

Comment:
To keep from either having everything in the authentication module, or having circular dependency issues, I split the component out of the authentication module and into a new user-account-actions module.